packages/terra-collapsible-menu-view/src/CollapsibleMenuView.jsx

```diff
@@ -91,11 +91,23 @@ class CollapsibleMenuView extends React.Component {
   render() {
     const { children, boundingRef, menuWidth, ...customProps } = this.props;
     const visibleChildren = React.Children.toArray(children);
+
     let hiddenChildren = null;
+    let singleHiddenIconChild = false;
     if (this.state.hiddenStartIndex >= 0) {
       hiddenChildren = visibleChildren.splice(this.state.hiddenStartIndex);
+      singleHiddenIconChild = hiddenChildren.length === 1 && hiddenChildren[0].props.isIconOnly;
```

Comment:
We shouldn't be basing this logic on the isIconOnly prop. We should just update with width calculations to determine if there is enough room for the last item if the menu is hidden.

Reply:
That would be better. I updated the changes.